packages/admin/admin/src/mui/menu/ItemGroup.tsx

```diff
@@ -76,7 +76,25 @@ const ItemGroup: React.FC<React.PropsWithChildren<WithStyles<typeof styles> & Me
     }
 
     if (!menuOpen) {
-        displayedTitle = shortTitle || getInitials(title);
```

Comment:
I'd revert this: If a developer decides to make short title longer than three letters, it's their problem 🤷🏼‍♂️